tools/spanner/purge_ttl.py

```diff
@@ -190,6 +194,14 @@ def get_args():
         help="Array of strings used to limit purges based on UID. "
              "Each entry is a separate purge run."
     )
+    parser.add_argument(
```

Comment:
Is it worth adding a guard to error out if both auto_split and uid_prefixes are specified?

Reply:
If this was a general use tool, absolutely. Since it's in-house, I'm willing to let it slide and just answer any questions SRE might come up with. 

If they hit it as a problem, I'll add the guards.